config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -120,6 +120,49 @@ periodics:
     testgrid-alert-email: skclark@redhat.com
     description: "Uses kubetest to run serial node-e2e tests (+Serial, -Flaky|Benchmark|Node*Feature)"
 
+- name: ci-kubernetes-node-arm64-ubuntu-serial
+  interval: 4h
+  cluster: k8s-infra-prow-build
+  annotations:
+    testgrid-dashboards: sig-node-kubelet
+    testgrid-tab-name: kubelet-gce-e2e-arm64-ubuntu-serial
+    description: "Run serial node e2e tests on ARM64 environment on Ubuntu"
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230513-7e1db2f1bb-master
+        args:
+          - --repo=k8s.io/kubernetes=master
```

Comment:
we should stop writing new jobs with bootstrap.py instead of `decorate: true`

these jobs have been logging a warning to migrate for _years_

this doesn't have to block adding the job, but please seriously consider migrating.

https://gist.github.com/dims/c1296f8ed42238baea0a5fcae45f4cf4

Reply:
Acknowledge, thanks for the gist pointer, will follow up in a separate PR.